pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -1270,17 +1276,17 @@ var _ = Describe("v1beta1 inference service controller", func() {
 	Context("When creating inference service with raw kube predictor and empty ingressClassName", func() {
 		configs := map[string]string{
 			"explainers": `{
-               "alibi": {
-                  "image": "kfserving/alibi-explainer",
+	             "alibi": {
+	                "image": "kfserving/alibi-explainer",
 			      "defaultImageVersion": "latest"
-               }
-            }`,
+	             }
+	          }`,
 			"ingress": `{
-               "ingressGateway": "knative-serving/knative-ingress-gateway",
-               "localGateway": "knative-serving/knative-local-gateway",
-               "localGatewayService": "knative-local-gateway.istio-system.svc.cluster.local",
-               "ingressDomain": "example.com"
-            }`,
+	             "ingressGateway": "knative-serving/knative-ingress-gateway",
+	             "localGateway": "knative-serving/knative-local-gateway",
+	             "localGatewayService": "knative-local-gateway.istio-system.svc.cluster.local",
+	             "ingressDomain": "example.com"
+	          }`,
```

Comment:
are these needed?

Reply:
Are you referring to the indentation? If so, I didn't intend to change it; my IDE did that automatically. However, logically, it doesn't impact anything, so I believe it should be fine